src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: getCalendarWidth($field-size, $field-gap, $calendar-padding);
+  gap: $field-gap;
+  padding: $calendar-padding;
+  margin: 0 auto;
+
+  &__field {
+    cursor: pointer;
+    @include square($field-size);
+    background-color: $background-color;
+    box-sizing: border-box;
+    border: 1px solid $border-color;
+    font-size: 30px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: all 0.5s;
```

Comment:
Avoid setting transition to all, list all styles that you apply transition to. It's more readable and you know exactly what is affected by it.

Reply:
Replace 'all' for transform and background color only